libpyvinyl/Instrument.py

```diff
@@ -0,0 +1,62 @@
+"""
+:module Instrument: Module hosting the Instrument class
+"""
+
+from libpyvinyl.Parameters.Collections import InstrumentParameters
+from pathlib import Path
+
+
+class Instrument():
+    """An Instrument class"""
+    def __init__(self, name, calculators=None):
+        """An Instrument class
+
+        :param name: The name of this instrument
+        :type name: str
+        :param calculators: A collection of Calculator objects.
+        :type calculators: list
+        """
+        self.name = name
+        self.calculators = {}
```

Comment:
This is a bit confusing, as a user I would see that the calculators input need to take "A collection" of calculator objects, but the type is "list" and the the calculators object is initialised as a "dict".

Reply:
Thank you, Mads, well spotted! It was a list at the beginning, and I forgot to update it.